src/app/[locale]/(auth)/login/page.tsx

```diff
@@ -10,9 +17,32 @@ export default function LoginPage() {
   };
 
   return (
-    <main className="h-full w-full bg-gradient">
-      로그인페이지
-      <button onClick={handleGoBack}>뒤로가기</button>
+    <main className="h-full w-full dark-gradient">
+      <div className="flex items-center justify-end p-3 pb-0">
+        <VscChromeClose className="mt-[10px] h-6 w-6 scale-transition-105" onClick={handleGoBack} />
+      </div>
+      <Swiper
+        className="mt-[58px] h-[450px] w-[394px]"
+        modules={[Pagination]}
+        slidesPerView={1}
+        pagination={{ clickable: true }}>
+        <SwiperSlide className="flex items-center justify-center">
```

Comment:
```suggestion
        <SwiperSlide className="flex-center">
```
요거 어제 구두로 말씀드렸던 것 같은데, flex-center, flex-col-center 클래스 추가했으니 해당 클래스로 사용 부탁드립니다

Reply:
✅ 8916f28e0a62589b09af82bbc581c80930a9f2c0